module/users/helper.js

```diff
@@ -539,10 +543,25 @@ module.exports = class UsersHelper {
           programName: programData[0].name,
           programId: programId,
           description: constants.common.TARGETED_SOLUTION_TEXT,
+          requestForPIIConsent: programData[0].requestForPIIConsent ? programData[0].requestForPIIConsent : false,
           data: mergedData,
           count: totalCount
         };
 
+        //check programUsers collection for consentForPIIDataSharing
+        let programUsersData = await programUsersHelper.find(
+          {
+            programId: programId,
+            userId: userId
```

Comment:
@VISHNUDAS-tunerlabs the order of query has to be userId first and then programId